src/components/App.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
-import Display from './Display';
-import ButtonPanel from './ButtonPanel';
-
-const App = function app() {
-  return (
-    <>
-      <Display />
-      <ButtonPanel />
-    </>
-  );
-};
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please remove the disabled eslint for unused-vars as pointed out by the last two reviewers. This means that you have declared some variables that you have not used. You have imported `useState, Component, Fragment` from react without using them. Please, do not re-submit without fixing this.